xdrip-get-entries.sh

```diff
@@ -584,6 +585,17 @@ function check_pump_history_calibration()
   calDate=$epochdate # TODO: use pump history date
 }
 
+function check_variation()
+{
+  variation=$(bc <<< "($filtered - $unfiltered) * 100 / $filtered")
+  if [ $(bc <<< "$variation > 10") -eq 1 -o $(bc <<< "$variation < -10") -eq 1 ]; then
```

Comment:
I checked my g5.csv file and there were only about 1% of the time where deviation was greater than 10% and only one time at calibration and sure enough, that calibration should have been ignored, so I think this change is good, or at least a good starting point.

Reply:
yeah, it doesn't happen much for our use either, mainly at end of sensor life.  since calibration still goes to dexcom (this check is after that i believe & they should have their own constraints), i'm good with skipping potential bad cals.